src/util-ebpf.c

```diff
@@ -140,10 +158,129 @@ int EBPFGetMapFDByName(const char *iface, const char *name)
             return bpf_maps->array[i].fd;
         }
     }
+
+    /* Fallback by getting pinned maps ? */
```

Comment:
add TODO label?

Reply:
Nope, it is implemented an other way. Removing the line.